.github/dependabot.yml

```diff
@@ -3,8 +3,11 @@ updates:
   - package-ecosystem: gomod
     directory: /
     schedule:
-      interval: daily
-    open-pull-requests-limit: 5
+      interval: weekly
+      day: friday
```

Comment:
Friday is usually the day people take off, and dependency updates can break CI over the weekend, etc. Monday may be a better choice.